Dockerfile

```diff
@@ -17,6 +17,8 @@ RUN cd /opt && \
 # and hosts.
 RUN yum install -y nss_wrapper && yum clean all
 
+RUN yum install -y numpy && yum clean all
```

Comment:
heh, I intentionally made this a separate layer this time because matt in the past suggested that keeping the layers logically separate might give us a time saving later on push.

Difference is 17MB, I can go either way ... no strong opinion